### PR TITLE
Enterprise suite 0.0.18 (beta RC)

### PR DESCRIPTION
**Pull request by marcoderama**

### CODE REVIEW

File: docs/es/all-latest.yaml [hunk 1]
```diff
@@ -160,6 +158,7 @@ data:
       {"graphName":"Response Rate", "promQL":["sum by (request_path)(irate(akka_http_request_path_endpoint_responses{workload=\"$workloadId\"}[1m]))"]},
       {"graphName":"Response Time", "promQL":["max by (request_path)(akka_http_request_path_endpoint_response_time_ns{workload=\"$workloadId\"})"]}
     ]
+    e
```
Comment (katsutoxin) [inline docs/es/all-latest.yaml]: why there is "e"?

File: docs/es/all-latest.yaml [hunk 2]
```diff
@@ -132,18 +132,16 @@ data:
...
+      {"graphName":"Dispatcher Processing Time", "promQL":["sum by (actor_system) (akka_dispatcher_processing_time_ns{workload=\"$workloadId\"})"]},
+      {"graphName":"Dispatcher Queue Size", "promQL":["sum by (actor_system) (akka_dispatcher_queue_size{workload=\"$workloadId\"})"]},
+      {"graphName":"Dispatcher Processing", "promQL":["sum by (actor_system) (akka_dispatcher_processing{workload=\"$workloadId\"})"]},
+      {"graphName":"Dispatcher Pool Size", "promQL":["sum by (actor_system) (akka_dispatcher_pool_size{workload=\"$workloadId\"})"]},
+      {"graphName":"Dispatcher Active Threads", "promQL":["sum by (actor_system) (akka_dispatcher_active_threads{workload=\"$workloadId\"})"]},
+      {"graphName":"Dispatcher Running Threads", "promQL":["sum by (actor_system) (akka_dispatcher_running_threads{workload=\"$workloadId\"})"]},
+      {"graphName":"Dispatcher Parallelism", "promQL":["sum by (actor_system) (akka_dispatcher_parallelism{workload=\"$workloadId\"})"]}
     ]
-  akka-http-exporter.json: |
+  akka-http-exporter.json: |-
```
Comment (katsutoxin) [inline docs/es/all-latest.yaml]: why add a "-" ?

File: docs/es/all.yaml [hunk 3]
```diff
@@ -132,18 +132,16 @@ data:
...
+      {"graphName":"Dispatcher Processing Time", "promQL":["sum by (actor_system) (akka_dispatcher_processing_time_ns{workload=\"$workloadId\"})"]},
+      {"graphName":"Dispatcher Queue Size", "promQL":["sum by (actor_system) (akka_dispatcher_queue_size{workload=\"$workloadId\"})"]},
+      {"graphName":"Dispatcher Processing", "promQL":["sum by (actor_system) (akka_dispatcher_processing{workload=\"$workloadId\"})"]},
+      {"graphName":"Dispatcher Pool Size", "promQL":["sum by (actor_system) (akka_dispatcher_pool_size{workload=\"$workloadId\"})"]},
+      {"graphName":"Dispatcher Active Threads", "promQL":["sum by (actor_system) (akka_dispatcher_active_threads{workload=\"$workloadId\"})"]},
+      {"graphName":"Dispatcher Running Threads", "promQL":["sum by (actor_system) (akka_dispatcher_running_threads{workload=\"$workloadId\"})"]},
+      {"graphName":"Dispatcher Parallelism", "promQL":["sum by (actor_system) (akka_dispatcher_parallelism{workload=\"$workloadId\"})"]}
     ]
-  akka-http-exporter.json: |
+  akka-http-exporter.json: |-
```
Comment (katsutoxin) [inline docs/es/all.yaml]: why there is "-"

File: docs/es/all.yaml [hunk 4]
```diff
@@ -160,6 +158,7 @@ data:
       {"graphName":"Response Rate", "promQL":["sum by (request_path)(irate(akka_http_request_path_endpoint_responses{workload=\"$workloadId\"}[1m]))"]},
       {"graphName":"Response Time", "promQL":["max by (request_path)(akka_http_request_path_endpoint_response_time_ns{workload=\"$workloadId\"})"]}
     ]
+    e
```
Comment (katsutoxin) [inline docs/es/all.yaml]: why there is "e"

File: enterprise-suite/values.yaml [hunk 5]
```diff
@@ -1,13 +1,14 @@
 # Update these versions for a release.
 
 # lightbend-docker-registry.bintray.io/enterprise-suite/es-console
-esConsoleVersion: 0.0.8
+esConsoleVersion: 0.0.9
 # lightbend-docker-registry.bintray.io/enterprise-suite/es-monitor-api
-esMonitorVersion: 0.0.2
+#esMonitorVersion: 0.0.2
```
Comment (katsutoxin) [inline enterprise-suite/values.yaml]: remove this line

File: docs/es/all-latest.yaml (hunk 1 above)
Comment (marcoderama) [reply docs/es/all-latest.yaml]: No idea.  Looks wrong for sure

**Pull request merged**